src/main/kotlin/no/nav/syfo/unntak/api/UnntakApi.kt

```diff
@@ -55,8 +57,26 @@ fun Route.registerUnntakApi(
                 call.respond(unntakDTOList)
             }
         }
-        get(unntakApiHackaton) {
-            call.respond(unntakService.getHackaton(call.getNAVIdent()))
+        get(unntakApiStatistikk) {
+            val unntakForventetFriskmelding = unntakService.getUnntakForVeileder(call.getNAVIdent()).filter { unntak ->
+                unntak.arsak == UnntakArsak.FORVENTET_FRISKMELDING_INNEN_28UKER
+            }
+            val unntakStatistikkDTOList = unntakForventetFriskmelding.mapNotNull {
+                val unntakDato = it.createdAt.toLocalDateTimeOslo().toLocalDate()
+                val tilfelleForUnntak = oppfolgingstilfelleService.getOppfolgingstilfelleForDate(
```

Comment:
Dersom det finnes flere unntak for samme person så vil det bli flere kall til isoppfolgingstilfelle for å hente oppfølgingstilfelle her. Kunne kanskje optimalisert med å hente (og ta vare på) alle oppfolgingstilfeller og finne det som gjelder unntak-dato. Innspill?

Reply:
Hvorfor flyttet du denne logikken fra `UnntakService` og hit? Jeg liker best at api-filen er liten, og service-filer har mer logikk 🤔

Reply:
Vil ikke en cache funke greit?

Reply:
Tenker at `UnntakService` ikke bør ha avhengighet til API-DTO-er, men bare `Unntak` eller andre domene-konsepter. Og så kan man bruke `OppfolgingstilfelleService` direkte for å finne tilfeller, i stedet for å gå via `DialomotekandidatService`. Man for så vidt enig i at det er fint med tynne API-filer... kanskje vi bør ha en UnntakStatistikkService eller en domeneklasse for UnntakStatistikk 🤔

Reply:
Jo, tenkte egentlig bare å holde på en Map<PersonIdent, List<Oppfolgingstilfelle>her for det aktuelle kallet. Eller tenker  du Redis?

Reply:
Jeg tenkte egentlig på Redis, men det er jo ikke noe krav å bruke det! Hvis du enkelt kan løse det med en map, er det helt kurant 👍 Redis vil hjelpe hvis flere veiledere har satt unntak på samme innbygger, men det kan vel ikke gjelde så veldig mange? 🤔

Reply:
Nei, det er vel neppe veldig vanlig... 
Ser om jeg kan få til en enkel cache i ny PR.